From 61aa245a0c4a2bffa498138139b822c227a91a34 Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Mon, 28 Oct 2024 22:26:20 +0100
Subject: [PATCH] main.rs: adjust application_id

This fixes icon not shown in applications that rely on proper application_id
naming (e.g. nwg-dock-hyprland)

Signed-off-by: Markus Volk <f_l_k@t-online.de>
---
 src/main.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.rs b/src/main.rs
index 9bc13ce..4d39674 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -151,7 +151,7 @@ along with this program; if not, see
     }
 
     let app = Application::builder()
-        .application_id("nnyyxxxx.hyprwall")
+        .application_id("hyprwall")
         .build();
 
     app.connect_activate(gui::build_ui);
-- 
2.47.0

